api/client/client.go

```diff
@@ -1319,6 +1319,52 @@ func (c *Client) DeleteAllNodes(ctx context.Context, namespace string) error {
 	return trail.FromGRPC(err)
 }
 
+// StreamSessionEvents streams audit events from a given session recording.
+func (c *Client) StreamSessionEvents(ctx context.Context, sessionID string, startIndex int) (chan error, chan events.AuditEvent) {
```

Comment:
nit: up to you if you want to change this, but I would expect the `chan error` return to be second just because `error` returns are usually second

Reply:
Good point, just swapped out the context and didn't think about the ordering. Adjusted